packages/backend/src/api/topic.ts

```diff
@@ -14,10 +13,22 @@ export class Topic extends BaseModel {
   name: string;
 }
 
-@RegisterCrudApi(app, { model: Topic, route: "/topic", memorySize: 512 })
+@CrudApi({ model: Topic, route: "/topic", memorySize: 512 })
```

Comment:
How do we stick schema validation on top of it?

Use that library to put decorators on top of fields in `Topic`?
https://github.com/typestack/class-validator

Reply:
Yeah that's one option and would fit nicely. Would like to look into io-ts again

Reply:
There's a library mainly for form validation, that allows to create a schema and then validate user input against it.

If we can stick it on top of the API could kill 2 birds with 1 stone.

https://github.com/jquense/yup